spec/spec-helper.rb

```diff
@@ -0,0 +1,14 @@
+# Requires supporting ruby files with custom matchers and macros, etc,
+# in spec/support/ and its subdirectories.
+Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each { |f| require f }
```

Comment:
This won't match any files right now. Unless you have a need for it in an imminent PR, you can just remove this line.